tests/test_integration/test_commands/test_profile.py

```diff
@@ -6,9 +6,19 @@ class TestDumpResource:
     def test_profile_assent_centric(self, toolkit_client: ToolkitClient, monkeypatch) -> None:
         results = ProfileCommand().asset_centric(toolkit_client, verbose=False)
 
-        assert len(results) == 5
-        assert {item["Resource"] for item in results} == {"Assets", "Events", "Files", "TimeSeries", "Sequences"}
+        assert len(results) == 7
+        assert {item["Resource"] for item in results} == {
+            "Assets",
+            "Events",
+            "Files",
+            "TimeSeries",
+            "Sequences",
+            "Relationships",
+            "Labels",
+        }
         total_count = sum(item["Count"] for item in results)
         assert total_count > 0
```

Comment:
Is it possible to have a slightly stronger assertion here? "There is something asset-centric in the project" is a bit thin.

Reply:
Do you have anything in mind? 

The intention of this test is to ensure that the asset-centric profiling runs end-to-end and does not raise any exceptions. Each individual part is tested more accurately as you see in the test below. Thus I prefer to keep it as is.